src/pages/Mypage/MyPage.js

```diff
@@ -0,0 +1,119 @@
+import { React, useState } from 'react';
+import * as S from './myPageStyle';
+
+const MyPage = () => {
+  const [visible, setVisible] = useState(false);
+
+  return (
+    <div className="myPage">
+      <S.MyPageLayout>
+        <S.Profile>
+          <S.ImgDiv>
+            <S.ImgBorder>
+              <S.ProFileImg
+                src="https://studiomeal.com/wp-content/uploads/2020/01/01-00.jpg"
+                alt="wow"
+              />
+            </S.ImgBorder>
+          </S.ImgDiv>
+          <S.NickName>봉봉</S.NickName>
+          <S.KakaoId>@bongbong</S.KakaoId>
+          <S.Following>팔로잉 3명</S.Following>
+          <S.Btn>
+            <S.Button>프로필 수정</S.Button>
+            <S.Button>팔로우 하기</S.Button>
+          </S.Btn>
+        </S.Profile>
+        <S.MiddleDiv>
+          <S.MyBoard>저장됨</S.MyBoard>
+        </S.MiddleDiv>
+        <S.MakingPin>
+          <S.MakingPinBtnParent>
+            <S.MakingPinBtn />
+            <S.MakingPinBtnHover />
+            {/* 보드 만드는 모달로 가는 버튼 */}
+          </S.MakingPinBtnParent>
+        </S.MakingPin>
+        <S.BoardList>
+          <S.Board
+            onClick={() => {
+              setVisible(true);
+            }}
```

Comment:
```suggestion
            onClick={() => setVisible(true)}
```
화살표 함수에서 한줄이라면 {} 를 생략할 수 있습니다.
취향차이지만 저는 없는 게 더 깔끔해 보일 것 같아요!